pkg/api/validation/validation.go

```diff
@@ -419,6 +423,17 @@ func validateGlusterfs(glusterfs *api.GlusterfsVolumeSource) errs.ValidationErro
 	return allErrs
 }
 
+func validateCinderVolumeSource(PD *api.CinderVolumeSource) errs.ValidationErrorList {
```

Comment:
If you can turn this PR around in a day or two I can review it again before
the real feature freeze

On Mon, Jun 1, 2015 at 5:19 PM, spothanis <notifications@github.com> wrote:

> In pkg/api/validation/validation.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/6689#discussion_r31484207>
> :
>
> > @@ -419,6 +423,17 @@ func validateGlusterfs(glusterfs *api.GlusterfsVolumeSource) errs.ValidationErro
> >  	return allErrs
> >  }
> >
> > +func validateCinderVolumeSource(PD *api.CinderVolumeSource) errs.ValidationErrorList {
>
> ahh, this was an over sight, thanks for catching it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/6689/files#r31484207>
> .
>